src/SamplesApp/SamplesApp.UITests/Helpers/UITestHelper.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using SamplesApp.UITests;
+
+namespace UITests.Shared.Helpers
+{
+	public static class UITestHelper
+	{
```

Comment:
This tooling already exists here: https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.UITests/TestFramework/ImageAssert.cs, please use this instead.